lvsfunc/misc.py

```diff
@@ -408,6 +410,53 @@ def upscale_chroma(n: int, f: vs.VideoFrame) -> vs.VideoNode:
     return cast(F, inner)
 
 
+def colored_clips(amount: int, max_hue: int = 300, **kwargs: Any) -> List[vs.VideoNode]:
+    """
+    Returns a list of BlankClips with randomized unique colors. Useful maybe for comparison functions
```

Comment:
This documentation seems to be inaccurate, as it produces evenly spaced hues in a randomized order